### PR TITLE
Incorrect Waterlog info fix

### LINKED ISSUE #1155: Incorrect Waterlog info
Something appears to be wonky with SLIMS and the GUI. it looks like for both of alex's mice, what should be the post session weight being entered is actually the baseline weight: 
image
WT entry is 29.2 (baseline weight) rather than ~23g (fluctuating depending on day).
Currently the gui is logging waterlogs with this logic
image
weight_g should be the post session weight

### PR DESCRIPTION
**Pull request by micahwoodard**
### Pull Request instructions:
- Please follow the [update protocol](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki/Software-Update-Procedures)
- Answer the questions below in detail. Your responses will be emailed to experimenters. 
- If the experimenters must do anything new, provide detailed step by step instructions on the [wiki](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki)
- If computer maintainers need to manually update anything, provide detailed step by step instructions
- Use markdown syntax in order for your comments to be rendered reliably in the email: "1." instead of "1)", use four spaces for indents.
- If you use the keyword "skip email" in the title, it will skip the email updates
- Merges from "develop" into "production_testing" should use the keyword "production merge" in the title for reliable indexing of updates
- Merges from "production_testing" into "main" should use the keyword "update main"
  
### Describe changes:
- waterlog now record weight_g as animal_weight_post and records operator of session
### What issues or discussions does this update address?
- resolves #1155 
### Describe the expected change in behavior from the perspective of the experimenter
- None
### Describe any manual update steps for task computers
- None
### Was this update tested in 446/447?
- [ ] 446/7

### CODE REVIEW

**Comment by micahwoodard** [comment]
@jessy-liao just added you to let you know that operator is being added

**Review by alexpiet — APPROVED** [review]
Thanks for the quick fix!

**Comment by micahwoodard** [comment]
@XX-Yin this patch should resolve [#799](https://github.com/AllenNeuralDynamics/aind-behavior-blog/issues/799). Pulling software for stimulus epoch instead of data_stream

**Review by XX-Yin — APPROVED** [review]

**Comment by XX-Yin** [comment]
We probably need to merge the develop to the main after fixing https://github.com/AllenNeuralDynamics/aind-behavior-blog/issues/799#issuecomment-2537395989 later of today.

**Comment by micahwoodard** [comment]
Yeah @alexpiet any opinions on how we handle this? Since production_testing merged with main yesterday, only things merged into develop will be #1156 and whatever fixes @XX-Yin makes. Merging develop into production_testing and then production_testing to main once tested seems okay to me.

**Comment by XX-Yin** [comment]
> Yeah @alexpiet any opinions on how we handle this? Since production_testing merged with main yesterday, only things merged into develop will be #1156 and whatever fixes @XX-Yin makes. Merging develop into production_testing and then production_testing to main once tested seems okay to me.

That sounds good to me.

**Comment by micahwoodard** [comment]
@XX-Yin actually it looks like quite a few merges were made since the last production_testing merge. I think to limit the amount of new issues, we should just merge this branch and your fix into production testing and then production_testing into main. I can make those pull requests if you can approve and merge?

**Comment by micahwoodard** [comment]
> @XX-Yin actually it looks like quite a few merges were made since the last production_testing merge. I think to limit the amount of new issues, we should just merge this branch and your fix into production testing and then production_testing into main. I can make those pull requests if you can approve and merge? I think that will be fine in terms of merge errors? What do you think?

Ack I don't think there is a way around merging in the other branches since the other two branches were branched off of development. I guess the only option to fix is to merge develop into production_testing and main.

File: tash [hunk 1]
```diff
@@ -0,0 +1,31 @@
+[1mdiff --git a/src/foraging_gui/Foraging.py b/src/foraging_gui/Foraging.py[m
```
Comment (XX-Yin) [inline tash:1]: What is the tash file?

Comment (XX-Yin) [reply tash:1]: After talking with Micah, this file seems to be added by mistake. @micahwoodard Could you make a PR to delete it?

**Pull request merged**